Testreglar/3.3.2/App/app-3.3.2a.json

```diff
@@ -1,378 +1,484 @@
+								}
+							}
+						},
+						"2": {
+							"sjekk": "2.5",
+							"type": "lik",
+							"verdi": "Ja",
+							"handling": {
+								"type": "avslutt",
+								"fasit": "Ja",
+								"utfall": "Inndataelement har identifikasjon i form av ikon/symbol/bilete. Skjema har tekstleg informasjon om at alle inndataelementer er obligatoriske."
+							}
+						}
+					}
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Inndataelement med identifikasjon i form av: #delutfall(0,Nei)<br>- ikon/symbol/bilde, som ikkje er identifisert i koden."
+				}
+			}
+		},
+		{
+			"stegnr": "3.9",
+			"spm": "Er det aktuelle inndataelementet obligatorisk?",
+			"ht": "<p>Obligatorisk inneber at brukaren må fylle ut informasjon eller gjere valg, for å kome vidare i skjemaet, for eksempel</p>\n<ul>\n<li>fylle ut informasjon </li>\n<li>gjere val i nedtrekksmenyar, radioknappar, avkryssingsboksar.</li>\n</ul>\n<p><strong>Merk:</strong> Du må kanskje fylle ut eller prøve å sende inn skjemaet for å finne ut om inndataelementet er obligatorisk.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Inndataelement med #delutfall(1,Ja)."
+				}
+			}
+		},
+		{
+			"stegnr": "3.10",
+			"spm": "Inneheld ledeteksten tekstleg informasjon om at inndataelementet er obligatorisk?",
+			"ht": "<p>Eksempel på tekstleg informasjon er</p>\n<ul>\n<li>obligatorisk</li>\n<li>påkrevd</li>\n<li>må fyllast ut</li>\n</ul>\n<p>Du skal ikkje måtte scrolle bort frå inndataelementet for å finne informasjonen.</p>\n<p><strong>Merk:</strong> Eventuell feilmelding, som dukkar opp <strong>etter </strong>at eit obligatorisk inndataelement ikkje er fylt ut, er ikkje tilstrekkeleg. Brukaren skal få informasjon på førehand. </p>",
+			"type": "jaNei",
+			"kilde": [
+				"G83",
+				"H90"
+			],
+			"ruting": {
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Inndataelement med #delutfall(1,Ja). Informasjon om obligatoriske inndataelement er tekst."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.11"
+				}
+			}
+		},
+		{
+			"stegnr": "3.11",
+			"spm": "Inneheld ledeteksten symbol/ikon som viser at inndataelementet er obligatorisk?",
+			"ht": "<p>Eksempel på symbol/ikon er stjerne (*)</p>\n<p><strong>Merk:</strong> Informasjon i form av symbol/ikon som dukkar opp <strong>etter</strong> at eit obligatorisk inndataelementet ikkje er fylt ut, er ikkje tilstrekkeleg.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"H90"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.12"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Skjema har ikkje informasjon om at inndataelementer er obligatoriske."
+				}
+			}
+		},
+		{
+			"stegnr": "3.12",
+			"spm": "Er symbolet/ikonet tekstleg forklart?",
+			"ht": "",
```

Comment:
Steg 3.12 mangler hjelpetekst

Reply:
Treng det stå hjelpetekst i alle steg? Ellers manglar også eksisterande testreglel hjelpetekst for tilsvarande steg, og det ligg  ikkje spørsmål knytt til dette i testlab.